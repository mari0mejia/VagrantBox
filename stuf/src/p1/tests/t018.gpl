// The negative and plus signs have their own tokens, T_MINUS and T_PLUS.  
// If there is a syntax error, then the cause is likely that the regular
// expression for T_INT_CONSTANT and/or T_DOUBLE_CONSTANT incorrectly captures
// the leading + or -.
double somedouble;
int    someint;

double negative = somedouble-98.7;
int    nosiree  = someint-24;

int    positive = negative+32;
double affirmitive = nosiree+3.14;
