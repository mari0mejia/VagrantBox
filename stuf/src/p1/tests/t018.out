gpl.cpp::main()
  input file(tests/t018.gpl)
  random seed(42)
  read_keypresses_from_standard_input(false)
  dump_pixels(false)
  symbol_table(false)
  graphics(false)

gpl.cpp::main() Calling yyparse()

Double constant -->98.7<--
Integer constant -->24<--
Integer constant -->32<--
Double constant -->3.14<--

gpl.cpp::main() after call to yyparse().

No errors found (parser probably worked correctly).

Graphics is turned off by the Makefile.  Program exiting.

gpl.cpp::main() done.
